edgebuilder-install.sh

```diff
@@ -570,9 +549,11 @@ display_usage()
   echo "     -u, --uninstall          : Uninstall the package"
   echo "     -f, --file               : Absolute path to local package"
   echo "     --offline-provision      : Enable offline node provision"
+  echo "     --install-docker         : Install docker as part of package install"
```

Comment:
Add >&3 to the end of each line to output

Reply:
done